assets/_scss/_users-module.scss

```diff
@@ -186,3 +187,27 @@
 .RadioButtonGroup [type="radio"]:nth-of-type(3):checked ~ .RadioButtonGroup__Details *:nth-of-type(3) {
   display: block;
 }
+
+// Floating Action Button
+
+.Fab {
```

Comment:
Does this need to be a separate button? Can we use the same 'Add' button that's being used on desktop, just with a media query?

Also, the name `Fab` is kinda ambiguous and not really in the convention of any of your other class names. Another developer digging into your code and stumbling upon the class name 'Fab' is gonna be like 'lol wut.'

Reply:
I could approach it that way, I think utility classes would make it convoluted, so I would probably opt to add a class name, then add a media query for that class and add those css rules to it. I'll also have to include add the svg markup to the Add Button and display none on larger screen sizes. Thinking out loud it feels a little messy, but easily doable.


also: 
https://www.google.com/search?q=fab+button&oq=fab+button&aqs=chrome..69i57j0l2j69i60l3.5040j0j0&sourceid=chrome&ie=UTF-8

but you're right not everyone may be versed on that.